config/locales/de.yml

```diff
@@ -7,19 +7,20 @@ de:
   verification_expired:
   feed_latest: RubyGems.org | Neueste Gems
   feed_subscribed: RubyGems.org | Abonnierte Gems
-  footer_about_html:
```

Comment:
Since bin/fill_locales wants to make a bunch of changes, could you submit a separate pr with just that?